net/dns/manager_linux.go

```diff
@@ -40,6 +44,11 @@ func NewOSConfigurator(logf logger.Logf, interfaceName string) (ret OSConfigurat
 	if err != nil {
 		return nil, err
 	}
+	publishOnce.Do(func() {
+		m := clientmetric.NewGauge(fmt.Sprintf("linux_dns_mode_%s", mode))
```

Comment:
Perhaps `dns_manager_linux_mode_` to keep this in the dns_* namespace?

Reply:
Done.